test/adapter/json/nested_relationships_test.rb

```diff
@@ -0,0 +1,203 @@
+require 'test_helper'
+
+NestedPostSerializer = Class.new(ActiveModel::Serializer) do
+  attributes :id, :title
+
+  has_many :comments, include: :author
+end
```

Comment:
I'd prefer normal class definitions.
I don't know why `Class.new(...)` is used though. :-\

I was just following existing patterns.